content/en/docs/Badges/MCH2022/software-development/fpga/_index.md

```diff
@@ -1,20 +1,32 @@
 ---
-title: "Getting Started with FPGA Development"
+title: "FPGA Development"
 linkTitle: "FPGA"
 nodateline: true
 weight: 1
 ---
 
-# FPGA development
+## TL;DR
 
-The badge contains an ice40 FPGA that is connected to a PMOD connector, some
-RAM, and an RGB LED.  It can also control the display over a parallel bus, and
-has an USB UART link via the RP2040 and an SPI link to the ESP32. See here for
-a confusing [diagram with boxes and arrows](../../hardware) showing how everything fits together.
+```
+git clone https://github.com/badgeteam/mch2022-tools/
+git clone --recursive https://github.com/badgeteam/mch2022-firmware-ice40
+python3 mch2022-tools/webusb_fpga.py mch2022-firmware-ice40/projects/Hello-World/hello_world.bin
+```
 
-Check out our ["FPGA **Getting Started** for Badgers with Tiny
-Brains"](fpga-getting-started) guide for step by step information getting from
-0 to a hardly working FPGA setup, in case you never heard of FPGAs before.
+## Welcome
+
+The badge contains an ice40 FPGA that is connected to a PMOD connector, a serial QSPI
+RAM, and a RGB LED.  It can also control the display over a parallel bus, and
+has an USB UART link via the RP2040 and an SPI link to the ESP32 which notifies the FPGA
+on the state of the buttons and offers read access to large data files.
+
+You can start with having a look at the
+[top-level diagram of the badge hardware](../../hardware) of the complete badge,
+then proceed to the  [schematic](https://github.com/badgeteam/mch2022-badge-hardware/blob/master/schematic.pdf)
+and [pin constraints file](https://github.com/badgeteam/mch2022-firmware-ice40/blob/master/projects/_common/data/mch2022-proto4.pcf).
+
+
+## Quickstart
 
 As with all the other methods to program the badge, step one is to download
```

Comment:
Just a thought: maybe we can encourage people to just clone instead of downloading the ZIP ... Also: tools is included as a submodule to the ice-firmware ... we could direct people to use that, or remove it as a submodule  ...  

(this is independant of this review, though.

Reply:
Maybe a good idea; but tools do not change as quickly as the firmware itself. I left that as-is, feel free to change later.